From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
Date: Sat, 10 Oct 2026 20:18:56 +0200
Subject: [PATCH] Add a pregenerated libversion.h

The libnds Makefile generates it, with the version it sets.
---
 include/nds/libversion.h | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 include/nds/libversion.h

diff --git a/include/nds/libversion.h b/include/nds/libversion.h
new file mode 100644
index 0000000..b1bf04c
--- /dev/null
+++ b/include/nds/libversion.h
@@ -0,0 +1,10 @@
+#ifndef __LIBNDSVERSION_H__
+#define __LIBNDSVERSION_H__
+
+#define _LIBNDS_MAJOR_	2
+#define _LIBNDS_MINOR_	0
+#define _LIBNDS_PATCH_	0
+
+#define _LIBNDS_STRING "libNDS Release 2.0.0"
+
+#endif // __LIBNDSVERSION_H__
